### DISCUSSION TITLE
How to edit / enable only a specific LayerGroup with edit control?

### DISCUSSION THREAD

**Discussion by CityBelfast** [geoman-io/leaflet-geoman#853] [Questions & Help]
Hi, I couldn't find any options in leaflet.pm to assign editable layer control  into a specific Layer Group like leaflet.Draw doing. see the below example.
```
var drawLayerGroup = L.featureGroup().addTo(map);
var options ={
position:'topright',
 edit:{ featureGroup: drawLayerGroup  , remove:false} // <--------
}
var drawControl = new L.Control.Draw(options);
map.addControl(drawControl );
```
So, Is there any way I can achieve the above code using leaflet.pm library?

**Comment by Falke-Design** [comment]
You can set the layergroup to edit mode directly:
```
drawLayerGroup.pm.enable()
```

With a Toolbar button:
```
map.pm.addControls({editMode: false})

const editButton = {
    name: 'editgroup',
    title: L.PM.Utils.getTranslation('buttonTitles.editButton'),
    className: 'control-icon leaflet-pm-icon-edit',
    onClick: () => { },
    afterClick: (e, obj) => {
      if(obj.button._button.toggleStatus){
        drawLayerGroup.pm.enable();
      }else{
        drawLayerGroup.pm.disable();
      }
    },
    doToggle: true,
    toggleStatus: false,
    disableOtherButtons: true,
    block: 'edit',
    actions: ['finishMode'],
  };

  // add a new custom control
  map.pm.Toolbar.createCustomControl(editButton)
```

**Comment by CityBelfast** [comment]
Sorry, I'm getting an error due to undefined .getTranslation method (see the below error). I'm just wondering whether  it is something related to free version! can we create Custom EditLayer Control using the free version of leaflet.pm?
![image](https://user-images.githubusercontent.com/76213912/102775035-ff6c5a80-4383-11eb-957c-200b373d0383.png)

**Comment by Falke-Design** [comment]
It looks like, that are you not using the latest release. You can check this in the console with `L.PM.version` it should be 2.8.0

Maybe you are not importing the correct package. The project changed the name from leaflet.pm to leaflet-geoman. So please check if your are using the corret package while importing

**Comment by CityBelfast** [comment]
Yes, It was my mistake. i was using leaflet.pm version
![image](https://user-images.githubusercontent.com/76213912/102781329-69d6c800-438f-11eb-8b49-db94ba372841.png)
